src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx

```diff
@@ -4,14 +4,7 @@ import { getChildPageNodes } from '@/utils/getChildPageNodes';
 export const meta = {
   title: 'Server-Side Rendering',
   description: 'Use Amplify categories APIs from Next.js server-side runtimes.',
-  platforms: [
-    'javascript',
-    'react-native',
-    'react',
-    'nextjs',
-    'angular',
-    'vue'
-  ],
+  platforms: ['javascript', 'react', 'nextjs', 'angular', 'vue'],
```

Comment:
This page seems specific to next and showcases usage of `adapter-nextjs`. 
Should we just display this page for javascript and nextjs. Maybe react ?

Reply:
yes that makes sense as the current content of SSR is Next.js specific. Wdyt @renebrandel ?